docs/docs/developers/page-editor/index.md

```diff
@@ -9,6 +9,20 @@ content and configuration to the _components_.
 
 ![Page Editor](images/page-editor-overview.png)
 
+## Component actions
+
+When a component is selected the list of available actions is presented in the toolbar.
+Available actions list:
+
+| Title         | Name (technical ID)               | Description                                      |
```

Comment:
Just leave the `Name` as the column name. You can mention that name is ID in the description above.

Reply:
done